test/api/test_module_construct.cpp

```diff
@@ -8,18 +8,16 @@ TEST_CASE(add_op)
     migraphx::program p;
     migraphx::module m = p.get_main_module();
     migraphx::shape param_shape{migraphx_shape_float_type, {3, 3}};
-    auto x      = m.add_parameter("x", param_shape);
-    auto y      = m.add_parameter("y", param_shape);
+    std::vector<float> x_values(9, 1);
```

Comment:
I would prefer we rename this test to `add_literals` so that it's clear why the test was modified.

Reply:
Agreed. Changed named.